node-email-experiment/a-wrapped-module.js

```diff
@@ -0,0 +1,32 @@
+
+'use strict';
+const nodemailer = require('nodemailer');
+
+async function main() {
+
+  const transporter = nodemailer.createTransport({
+    host: 'smtp-mail.outlook.com',
+    secureConnection: false,
+    port: 587,
+    secure: false,
+    tls: { ciphers: 'SSLv3' },
+    auth: {
+      user: 'Steve.Demo.LearningFuze2@outlook.com',
+      pass: '02North05'
+    }
```

Comment:
Not too sure if this email account is still active, but you should change the password on it, because it's publicly visible now.